kin/transactions.py

```diff
@@ -85,27 +86,17 @@ def __init__(self, raw_tx, kin_asset):
 class SimplifiedOperation:
     """Class to hold simplified info about a operation"""
 
-    def __init__(self, op_data, kin_asset):
+    def __init__(self, op_data):
         if isinstance(op_data, Payment):
             # Raise error if asset is not KIN or XLM
             if op_data.asset.type != 'native':
```

Comment:
maybe const `NATIVE_ASSET = 'native'` somewhere and import it all over the library